src/mongocxx/test/options/create_collection.cpp

```diff
@@ -13,6 +16,15 @@ TEST_CASE("create_collection", "[create_collection]") {
     options::create_collection cc;
 
     SECTION("Can be exported to a document") {
+        auto rule = builder::stream::document{}
+                    << "brain" << builder::stream::open_document << "$exists"
+                    << "true" << builder::stream::close_document << builder::stream::finalize;
```

Comment:
It's safer to do `{"$exists": true}` because both `{"$exists": "true"}` and `{"$exists": "false"}` do the same thing. The argument is cast to a bool, so even the empty string will be `true`.